packages/minimongo/matcher.js

```diff
@@ -1,3 +1,4 @@
+import { Decimal } from 'meteor/mongo-decimal'
```

Comment:
Unfortunately, you can't use ES6 imports for weak-linked packages. If you wouldn't have the mongo-decimal package installed, this will throw since it can't find the imported package.

The way to work with weak-linked packages is by using the Meteor's package system:
`const Decimal = Package['mongo-decimal']?.Decimal`

But then the instanceOf will throw since you would be checking against undefined.

So either you will have to check first that Decimal isn't undefined, or, since it's only for instanceof checks, you could do something like:
`const Decimal = Package['mongo-decimal']?.Decimal || class DecimalStub {}`

Reply:
Ok! I was fixed it like this

```
let Decimal = Package['mongo-decimal']?.Decimal;

// Due to {weak: true} Decimal dependency on client
// declare minimal Decimal functionality
if (!Decimal) {
  class Decimal {
    constructor(value) {
      this.value = value.toString();
    }
    minus(value) {
      let _value = Number(this.value) - Number(value);
      return new Decimal(_value);
    }
    toNumber() {
      return Number(this.value);
    }
  }
}
```